config/purposes/scrna_core_cell_extraction.yml

```diff
@@ -100,7 +100,12 @@ LRC TR Bank Seq:
       child_spare_tube_rack_purpose_name: LRC TR Bank Spare
       ancestor_stock_tube_purpose_name: LRC Blood Vac
   :presenter_class: Presenters::TubeRackPresenter
-  :state_changer_class: StateChangers::AutomaticTubeRackStateChanger
+  # This needs to be the AutomaticTubeRackStateChanger.
+  # But, the work-completion for the AutomaticTubeRackStateChanger is not yet
+  # fixed for TubeRacks (this needs to be done in Sequencescape). So, for now,
+  # we are using the TubeRackStateChanger for Manual Transfers.
+  # See: https://github.com/sanger/limber/issues/2074
+  :state_changer_class: StateChangers::TubeRackStateChanger
```

Comment:
I'm wondering what happens here in the integration test. I think it currently continues with a submission on the tubes, but if we've not closed off the cell extraction requests on those tubes...what happens?

also, may as well comment the work_completion_request_type lines if not got work completion yet.